assets/css/style.css

```diff
@@ -0,0 +1,288 @@
+  text-decoration: underline;
+}
+
+/******************************************************************************
+ Main Content Elements
+*******************************************************************************/
+main {
+  background: #fff;
+  border: 1px solid #ddd;
+  padding: 0 20px;
+}
+
+h1,
+main a {
+  color: #4aaaa5;
+}
+
+h1 {
+  border-bottom: 3px #ddd solid;
+  padding: 10px 0;
+}
+
+main img {
+  /* ISSUE: only rounding top, left corner (20200215). */
+  /* border-radius: 20px; */
+  float: left;
+  padding: 0 20px 20px 0;
+  width: 240px;
+}
+
+main p {
+  font-size: 20px;
+  line-height: 2em;
+}
+
+main figure img {
+  padding: 0;
+}
+
+main figure img,
+.photo-label {
+  width: 240px;
+}
+
+/******************************************************************************
+ Form Elements
+*******************************************************************************/
+main form input[type=text],
+main form input[type=email],
+main form textarea {
+  color: #ccc;
+  padding: 5px 10px;
+  width: 100%;
+}
+
+/* Form input "glow" compliments of CSS Tricks:
+   https://css-tricks.com/snippets/css/glowing-blue-input-highlights/ */
+main form input,
+main form textarea {
+  -webkit-transition: all 0.30s ease-in-out;
+  -moz-transition: all 0.30s ease-in-out;
+  -ms-transition: all 0.30s ease-in-out;
+  -o-transition: all 0.30s ease-in-out;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  line-height: 1.25em;
+  margin: 5px 1px 3px 0px;
+  outline: none;
+}
+
+main form input[type=text]:focus,
+main form input[type=email]:focus, 
+main form textarea:focus {
+  border: 1px solid rgba(74, 170, 165, 1);
+  color: #777;
+}
+ 
+main form input:focus, 
+main form textarea:focus,
+main form input[type=button]:hover,
+main form input[type=submit]:hover,
+main form input[type=reset]:hover {
+  box-shadow: 0 0 5px rgba(74, 170, 165, 1);
+}
+
+input[type=button], input[type=submit], input[type=reset] {
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+  padding: 16px 32px;
+  text-decoration: none;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
+main form input[type=button],
+main form input[type=submit],
+main form input[type=reset] {
+  background-color: #4aaaa5;
+  cursor: pointer;
+  margin-bottom: 20px;
+  padding: 5px 10px;
+  text-align: center;
+  text-decoration: none;
+}
+
+main form label {
+  display: block;
+  margin-top: 10px;
+}
+
+/******************************************************************************
+ Footer Element
+*******************************************************************************/
+footer {
+  background: #666;
+  border-top: 5px #4aaaa5 solid;
+  height: 50px;
+  width: 100%;
+}
+
+/******************************************************************************
+ Class & ID Elements
+*******************************************************************************/
+.header-container,
+.main-container,
+main,
+footer {
+  clear: both;
+  overflow: auto;
+}
+
+.header-container {
```

Comment:
Good job keeping these class and id names semantic. It's a good idea to write classes and ids that describe the context of the tag, **not** the style. e.g.:

Bad:

```html
<div class="big-border-red-box">...</div>
```

Good:

```html
<div class="call-to-action">...</div>
```